src/styles/blocks/card.css

```diff
@@ -0,0 +1,106 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #ffff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 200px;
+  padding: 32px 16px 16px ;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+  padding-bottom: 40px;
+}
+
+.card__name {
+  width: 166px;
```

Comment:
Remove the fixed width (166px), as the content should fit by default within its parent container.